### DISCUSSION TITLE
Interrupting FaceDescriptor

### DISCUSSION THREAD

**Discussion by MustafaCeyhan** [vladmandic/face-api#146] [Q&A]
I am using face-api.js in a project.  Sometimes it takes a long time description an image taken in the face descriptor part of face-api.js. I want to interrupt the model when I want or after a particular time if this face descriptor process does not finish. So while the process of describing an image is in progress, I want to close the model midway. Do you think something like this can be done?

**Comment by vladmandic** [comment]
what is your environment and which backend are you using?  
and how long is very long? it should be just a fraction of a second.

**Reply by MustafaCeyhan** [reply]
Actually like this. The application I'm developing is running on a computer with an average GPU and CPU. But I'm trying to see how long it will run on computers with different processing power. Sometimes this can take a very long time.

Face Description is just a part of my application that I developed. If this part takes too long, I want to stop the model midway. But I don't know how to stop the face description model while it is running.

**Reply by vladmandic** [reply]
you did not answer the question at all  
at a minimum is it browser or nodejs and are you using cpu, wasm, webgl or webgpu backend?  
that is absolute bare minimum information i need.  

also, you did not say how long is "too long"???  
again, if analyzing face descriptor takes more than just a fraction of second, there is something really bad in your configuration.  
and if it takes fraction of a second, you can choose what to do when its done - i don't see a point to interrupt it mid-way.
